Base/usr/share/man/man8/TelnetServer.md

```diff
@@ -0,0 +1,25 @@
+## Name
+
+TelnetServer - Serenity telnet server
+
+## Synopsis
+
+```**sh
+$ TelnetServer [options]
+```
+
+## Description
+
+`TelnetServer` is a basic telnet server for Serenity. By default, it
+runs on port 23 and provides a shell upon connection.
+
+## Options
+
+* `-p`: Choose different port for TelnetServer to attach to.
+* `-c`: Choose different program for TelnetServer to run on connection.
```

Comment:
Can it accept more args to the command, ex. `TelnetServer -c /bin/foo argv1 argv2`? Let's mention it either way.